cypress/integration/examples/cookies.spec.js

```diff
@@ -0,0 +1,77 @@
+/// <reference types="cypress" />
+
+context('Cookies', () => {
```

Comment:
Make sure to use the correct testing framework. The current code uses Cypress, while the task allowed Mocha or Jest.